package.json

```diff
@@ -26,15 +26,15 @@
     },
     "devDependencies": {
         "@blockly/eslint-config": "^2.0.0",
-        "eslint": "^6.8.0",
+        "eslint": "^7.15.0",
         "gh-pages": "^3.1.0",
         "gulp": "^4.0.2",
         "gulp-header": "^2.0.9",
         "js-green-licenses": "^1.1.0",
+        "json-to-pretty-yaml": "^1.2.2",
         "lerna": "^3.20.2",
         "rimraf": "^3.0.2",
-        "typescript": "^3.8.3",
-        "json-to-pretty-yaml": "^1.2.2"
```

Comment:
Why are you removing json-to-pretty-yaml. It's used in gulpfile.js

Reply:
I'm not--they just got alphabetized.

Reply:
Oh true.